From 02789309edfef3309658532e73e0119528bece4f Mon Sep 17 00:00:00 2001
From: Michal Vasilek <michal.vasilek@nic.cz>
Date: Fri, 1 Oct 2021 17:44:38 +0200
Subject: [PATCH] mvebu, mpc85xx: enable BFQ

BFQ is an IO scheduler which enables higher IO throughput on hard drives
on sequential workloads which is especially nice for NASes.

Signed-off-by: Michal Vasilek <michal.vasilek@nic.cz>
---
 target/linux/mpc85xx/config-5.4 | 1 +
 target/linux/mvebu/config-5.4   | 1 +
 2 files changed, 2 insertions(+)

diff --git a/target/linux/mpc85xx/config-5.4 b/target/linux/mpc85xx/config-5.4
index d6683dab35..2b837342a7 100644
--- a/target/linux/mpc85xx/config-5.4
+++ b/target/linux/mpc85xx/config-5.4
@@ -114,6 +114,7 @@ CONFIG_I2C_BOARDINFO=y
 CONFIG_I2C_MPC=y
 CONFIG_ILLEGAL_POINTER_VALUE=0
 CONFIG_INITRAMFS_SOURCE=""
+CONFIG_IOSCHED_BFQ=y
 CONFIG_IRQCHIP=y
 CONFIG_IRQ_DOMAIN=y
 CONFIG_IRQ_FORCED_THREADING=y
diff --git a/target/linux/mvebu/config-5.4 b/target/linux/mvebu/config-5.4
index ea73bafe23..ce67d1c657 100644
--- a/target/linux/mvebu/config-5.4
+++ b/target/linux/mvebu/config-5.4
@@ -281,6 +281,7 @@ CONFIG_I2C_MV64XXX=y
 # CONFIG_I2C_PXA is not set
 # CONFIG_I2C_PXA_SLAVE is not set
 CONFIG_INITRAMFS_SOURCE=""
+CONFIG_IOSCHED_BFQ=y
 CONFIG_IRQCHIP=y
 CONFIG_IRQ_DOMAIN=y
 CONFIG_IRQ_DOMAIN_HIERARCHY=y
-- 
2.33.0

